test/secret/docker-compose.yml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2020-2021 - Offen Authors <hioffen@posteo.de>
```

Comment:
Should the directory containing the test case be called `secrets` maybe? Not sure.

Reply:
Not sure, I saw more singular terms in the test folder, so used the same approach.